### PR TITLE
Chapter14

### PR DESCRIPTION
**Pull request by heimrih**
Following users - Chapter14

### CODE REVIEW

**Comment by heimrih** [comment]
ready

File: app/controllers/relationships_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    @user = User.find_by id: params[:followed_id]
+    current_user.follow @user
+    respond_to do |format|
+      format.html { redirect_to @user }
+      format.js
+    end
+  end
+
+  def destroy
+    @user = Relationship.find_by(id: params[:id]).followed
```
Comment (manhnt-0960) [inline app/controllers/relationships_controller.rb]: please handle case `Relationship.find_by(id: params[:id])` return nil

File: app/views/users/show_follow.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<% provide :title, @title %>
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      <%= gravatar_for @user %>
+      <h1><%= @user.name %></h1>
+      <span><%= link_to t("follow.profile"), @user %></span>
+      <span><b><%= t "follow.microposts" %></b> <%= @user.microposts.count %></span>
```
Comment (manhnt-0960) [inline app/views/users/show_follow.html.erb]: please use `size` instead of `count`

File: config/routes.rb [hunk 3]
```diff
@@ -9,9 +9,14 @@
     post "/login", to: "sessions#create"
     delete "/logout", to: "sessions#destroy", as: :logout
 
-    resources :users
+    resources :users do
+        member do
+            get :following, :followers
+        end
```
Comment (manhnt-0960) [inline config/routes.rb]: indent 2 spaces

**Comment by heimrih** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Comment by ngocvt-0484** [comment]
good job

**Pull request merged**